--- a/config-defaults.php
+++ b/config-defaults.php
@@ -44,7 +44,13 @@
 
 // FILE LOCATIONS
 
-$rooturl            =   "http://{$_SERVER['HTTP_HOST']}/limesurvey"; //The root web url for your limesurvey installation.
+// File Locations
+if (isset($cmd_install) && $cmd_install) {
+    $rooturl            =   "http://localhost/limesurvey"; // A safe Root URL if we're doing a cmd_install
+} else {
+    $rooturl            =   "http://{$_SERVER['HTTP_HOST']}/limesurvey"; // The root web url for your limesurvey installation (without a trailing slash).
+    // The double quotes (") are important.
+}
 
 $rootdir            =   dirname(__FILE__); // This is the physical disk location for your limesurvey installation. Normally you don't have to touch this setting.
 // If you use IIS then you MUST enter the complete rootdir e.g. : $rootDir="C:\Inetpub\wwwroot\limesurvey"!
--- a/common.php
+++ b/common.php
@@ -232,12 +232,12 @@
 
 
 //check if database exist and access is not via install script
-if (!$database_exists && (strcasecmp($slashlesspath,str_replace(array("\\", "/"), "", $homedir."install")) != 0)) {
+if (!$database_exists && !$cmd_install && (strcasecmp($slashlesspath,str_replace(array("\\", "/"), "", $homedir."install")) != 0)) {
     die ("<br/>The LimeSurvey database does not exist. Please run the <a href='$homeurl/install/index.php'>install script</a> to create the necessary database.");
 }
  
  // Check if the DB is up to date
-if ($dbexistsbutempty && (strcasecmp($slashlesspath,str_replace(array("\\", "/"), "", $homedir."install")) != 0)) {
+if ($dbexistsbutempty && !$cmd_install && (strcasecmp($slashlesspath,str_replace(array("\\", "/"), "", $homedir."install")) != 0)) {
     die ("<br />The LimeSurvey database does exist but it seems to be empty. Please run the <a href='$homeurl/install/index.php'>install script</a> to create the necessary tables.");
 }
 
--- a/config.php
+++ b/config.php
@@ -44,8 +44,12 @@
                                        // a database with other applications. Suggested prefix is 'lime_'
 
 // File Locations
-$rooturl            =   "http://{$_SERVER['HTTP_HOST']}/limesurvey"; // The root web url for your limesurvey installation (without a trailing slash).
-// The double quotes (") are important.
+if (isset($cmd_install) && $cmd_install) {
+    $rooturl            =   "http://localhost/limesurvey"; // A safe Root URL if we're doing a cmd_install
+} else {
+    $rooturl            =   "http://{$_SERVER['HTTP_HOST']}/limesurvey"; // The root web url for your limesurvey installation (without a trailing slash).
+    // The double quotes (") are important.
+}
 
 $rootdir            =   '/usr/share/limesurvey'; // On Debian, this is the physical disk location for your limesurvey installation. Normally you don't have to touch this
                                             // setting. 
