diff --git a/application/config/tcpdf.php b/application/config/tcpdf.php
index 3ec0d97..ee299bd 100644
--- a/application/config/tcpdf.php
+++ b/application/config/tcpdf.php
@@ -203,7 +203,7 @@
     * HTML <small> font size ratio
     ***********************************************************/
 
-    $tcpdf['page_font'] = 'freemono';
+    $tcpdf['page_font'] = 'DejaVuSansMono';
     $tcpdf['page_font_size'] = 9;
 
     $tcpdf['small_font_ratio'] = 2/3;
