--- a/config-defaults.php
+++ b/config-defaults.php
@@ -558,8 +558,8 @@
 
 
 $homedir                 = "$rootdir".DIRECTORY_SEPARATOR."admin";       // The directory path of the admin scripts
-$publicdir               = "$rootdir";                                   // The directory path of the public scripts
-$tempdir                 = "$rootdir".DIRECTORY_SEPARATOR."tmp";         // The directory path where LimeSurvey can store temporary files
+$publicdir               = "/var/lib/limesurvey";                                   // The directory path of the public scripts
+$tempdir                 = "$publicdir".DIRECTORY_SEPARATOR."tmp";         // The directory path where LimeSurvey can store temporary files
 $imagedir                = "$rootdir".DIRECTORY_SEPARATOR."images";      // The directory path of the image directory
 $uploaddir               = "$rootdir".DIRECTORY_SEPARATOR."upload";
 $standardtemplaterootdir = "$rootdir".DIRECTORY_SEPARATOR."templates";   // The directory path of the standard templates
