diff --git a/application/config/config-defaults.php b/application/config/config-defaults.php
index 1ae7d46..c949c84 100644
--- a/application/config/config-defaults.php
+++ b/application/config/config-defaults.php
@@ -596,9 +596,9 @@ $config['adminimageurl']           = $config['styleurl'].$config['admintheme'].'
 $config['adminstyleurl']           = $config['styleurl'].$config['admintheme'].'/';         // Location of button bar files for admin script
 
 
-$config['publicdir']               = $config['rootdir'];                                   // The directory path of the public scripts
+$config['publicdir']               = "/var/lib/limesurvey";                                   // The directory path of the public scripts
 $config['homedir']                 = $config['rootdir'];       // The directory path of the admin scripts
-$config['tempdir']                 = $config['rootdir'].DIRECTORY_SEPARATOR."tmp";         // The directory path where LimeSurvey can store temporary files
+$config['tempdir']                 = $config['publicdir'].DIRECTORY_SEPARATOR."tmp";         // The directory path where LimeSurvey can store temporary files
 $config['imagedir']                = $config['rootdir'].DIRECTORY_SEPARATOR."images";      // The directory path of the image directory
 $config['uploaddir']               = $config['rootdir'].DIRECTORY_SEPARATOR."upload";
 $config['standardtemplaterootdir'] = $config['rootdir'].DIRECTORY_SEPARATOR."templates";   // The directory path of the standard templates
